collection_requirements.rst

```diff
@@ -0,0 +1,149 @@
+*****************************
+Ansible Collections Checklist
+*****************************
+
+.. contents:: Topics
+
+Overview
+========
+This document is for maintainers of collections to provide them help, advice, and guidance on making sure their collections are correct.
+**Warning:** Subject to frequent updates
+       This is a "living document", expect it to change as we progress with the Collections work over the next few months.
+
+Feedback
+========
+
+As with any project it's very important that we get feedback from users, contributors and maintainers. We recognize that the move to Collections is a big change in how Ansible is developed and delivered.
+
+Please raise feedback by
+
+* Discussing in ``#ansible-community`` in Freenode IRC
+* Adding to the `Community Working Group IRC meeting <https://github.com/ansible/community/issues/539>`_
+* Creating `GitHub Issues <https://github.com/ansible-collections/overview/issues>`_ against this repo
+
+Keeping informed
+================
+
+Be sure you're subscribed to:
+* `Changes impacting Collections <https://github.com/ansible-collections/overview/issues/45>` to track changes that Collection maintainers should be aware of 
+* The Bullhorn, a newsletter for the Ansible developer community, `back issues and how to add content <https://github.com/ansible/community/issues/546>`_
+
+Why is this needed
+===================
+
+
+Repo structure
```

Comment:
Better avoid shortened name, so Repository.
sorry to nit-pick

Reply:
Good call, fixed all.